File: backend/src/dto/return.cabinet.data.dto.ts
import Lent from 'src/entities/lent.entity';
import CabinetStatusType from 'src/enums/cabinet.status.type.enum';
import LentType from 'src/enums/lent.type.enum';

/**
 * 캐비넷을 대여하기 전 캐비넷에 대한 최소 정보
 */
export class ReturnCabinetDataDto {
  status: CabinetStatusType; // 사물함의 상태
  lent_type: LentType; // 사물함의 타입
  lents: Lent[]; // 해당 사물함을 대여하고 있는 lent 배열
}

File: backend/src/main/java/org/ftclub/cabinet/dto/ReturnCabinetDataDto.java
package org.ftclub.cabinet.dto;

import lombok.*
@AllArgsConstructor
@NoArgsConstructor
@Data
public class ReturnCabinetDataDto {
    private CabinetStatus status;
    private LentType lentType;
    private LentHistory[] lentHistories; // 기존 Lent[] lents 였던 부분입니다.
}